backend/utils/printful.js

```diff
@@ -193,6 +194,50 @@ const fetchShippingEstimate = async (apiKey, data) => {
   }
 }
 
+const fetchTaxRates = async (apiKey, data) => {
+  if (!apiKey) {
+    return {
+      status: 500,
+      success: false,
+      message: 'Service Unavailable'
+    }
+  }
+
+  const apiAuth = Buffer.from(apiKey).toString('base64')
+
+  const { recipient } = data
+
+  const query = {
+    recipient: {
+      city: recipient.city,
+      country_code: recipient.countryCode,
+      state_code: recipient.provinceCode,
+      zip: recipient.zip
+    }
+  }
+
+  const shippingRatesResponse = await fetch(`${PrintfulURL}/tax/rates`, {
+    headers: {
+      'content-type': 'application/json',
+      authorization: `Basic ${apiAuth}`
+    },
+    credentials: 'include',
+    method: 'POST',
+    body: JSON.stringify(query)
+  })
+  const json = await shippingRatesResponse.json()
+  if (json.result && Number.isFinite(json.result.rate)) {
+    return {
+      success: true,
+      ...pick(json.result, ['rate', 'required'])
+    }
+  } else {
+    return {
+      success: false
```

Comment:
should we log an error? and maybe also capture in Sentry so that we get some visibility when/if this starts failing?